recommender/engine/training/autoencoders.py

```diff
@@ -16,7 +16,7 @@ def evaluate_autoencoder(model, dataloader, loss_function, device):
     model.eval()
     with torch.no_grad():
         for batch in dataloader:
-            features = batch[0].to(device)
+            features = batch[0].float().to(device)
```

Comment:
Shouldn't we be casting to float earlier, in the dataloader for example?

Reply:
this is test hotfix and it will be refactored in other issue due to issues prioritization